File: CafeTier2/src/main/java/group7/Grpc/dto/EventDto.java
package group7.Grpc.dto;

import lombok.Data;

import java.util.UUID;
@Data
public class EventDto {
    private UUID id;
    private String name;
    private String description;
    private UUID entertainerId;
    private UUID cafeOwnerId;
    private String entertainerName;
    private String cafeOwnerName;
    private String date;
    private String Status;
    private int availablePlaces;
}

File: Presentation tier/Cafe/dto/EventDto.cs
using System.Runtime.Serialization;
using System.Text.Json.Serialization;

namespace Cafe.dto;

public class EventDto{
    [JsonPropertyName("id")]
    public Guid Id { get; set; }
    [JsonPropertyName("name")]
    public string Name { get; set; }
    [JsonPropertyName("description")]
    public string Description { get; set; }
    [JsonPropertyName("entertainerId")]
    public Guid EntertainerId { get; set; }
    [JsonPropertyName("cafeOwnerId")]
    public Guid CafeOwnerId { get; set; }
    [JsonPropertyName("entertainerName")]
    public string EntertainerName { get; set; }
    [JsonPropertyName("cafeOwnerName")]
    public string CafeOwnerName { get; set; }
 /*   [JsonPropertyName("startDate")]
    public DateTime Start { get; set; }
    [JsonPropertyName("endDate")]
    public DateTime End { get; set; }
    [JsonPropertyName("availablePlaces")]*/
    
    public int AvailablePlaces { get; set; }
    [JsonPropertyName("status")]
    public string Status { get; set; }

}
